assets/src/scripts/components/hydrograph/index.js

```diff
@@ -1,213 +1,130 @@
+const { xScaleSelector, yScaleSelector } = require('./scales');
+const { Actions, configureStore } = require('./store');
 
 
 // Function that returns the left bounding point for a given chart point.
 const bisectDate = bisector(d => d.time).left;
 
 
-// Create a time formatting function from D3's timeFormat
-const formatTime = timeFormat('%c %Z');
-
-
-class Hydrograph {
-    /**
-     * @param {Array} data IV data as returned by models/getTimeseries
-     * @param {Array} calculated median for discharge
-     * @param {String} yLabel y-axis label
-     * @param {String} title for svg's title attribute
-     * @param {String} desc for svg's desc attribute
-     * @param {Node} element Dom node to insert
-     */
-    constructor({data=[], medianStats=[], yLabel='Data', title='', desc='', element}) {
-        this._yLabel = yLabel;
-        this._title = title;
-        this._desc = desc;
-        this._element = element;
-        this._tsData = {};
-        this._medianStatsData = medianStats;
-
-        if (data && data.length) {
-            this._tsData.current = data;
-            this._drawChart();
-        } else {
-            this._drawMessage('No data is available for this site.');
-        }
-    }
 
-    /**
-     * Add a new time series to the Hydrograph. The time series is assumed to be
-     * data that is over the same date range in a different year.
-     * @param {Array} data - IV data as returned by models.getTimeseires
-     */
-    addCompareTimeSeries(data) {
-        //Save data - will be needed in order to implement the tooltips
-        this._tsData.compare = data;
-
-        // Update the yScale by determining the new extent
-        const currentYExtent = extent(this._tsData.current.concat(this._medianStatsData), d => d.value);
-        const yExtent = extent(data, d => d.value);
-        const yDataExtent = [min([yExtent[0], currentYExtent[0]]), max([yExtent[1], currentYExtent[1]])];
-        updateYScale(this.scale.yScale, yDataExtent);
-
-        // Create a x scale for the new data
-        const xScale = createXScale(data, WIDTH - MARGIN.right);
-
-        // Update the yAxis
-        updateYAxis(this.axis.yAxis, this.scale.yScale);
-        this.svg.select('.y-axis')
-            .call(this.axis.yAxis);
-
-        //Update the current ts line
-        select('#ts-current')
-            .attr('d', this.currentLine(this._tsData.current));
-
-        //remove the median stats plot and recreate
-        this.svg.selectAll('#median-point').remove();
-        this.svg.selectAll('#median-text').remove();
-        this._plotMedianPoints();
-
-        // Add the new time series
-        this._plotDataLine(this.plot, {xScale: xScale, yScale: this.scale.yScale}, 'compare');
+const drawMessage = function (elem, message) {
+    // Set up parent element and SVG
+    elem.innerHTML = '';
+    const alertBox = elem
+        .append('div')
+            .attr('class', 'usa-alert usa-alert-warning')
```

Comment:
This really has nothing to do with this PR, but I think it would be more appropriate to use Information Status.